transmitter.js

```diff
@@ -21,11 +20,8 @@ function transmitInCelcius(farenheit) {
         // let us keep a count of failures to report
         // However, this code doesn't increment failures - that's a bug!
         // Add a test below to catch this bug. Alter the stub above, if needed.
-        transmissionFailureCount += 0;
+        transmissionFailureCount += 1;

```

Comment:
You have fixed the issue here... Fixing was part of the next assignment, not this one.
But still the expect is failing? Can you think why that may be happening?